.bazelversion

```diff
@@ -1 +1 @@
-5.1.1
+6.0.0-pre.20220414.2
```

Comment:
Do we need toolchain update for this or do we use bazilisk in build images?

Reply:
Based on [this log from CI](https://dev.azure.com/cncf/envoy/_build/results?buildId=107233&view=logs&j=8c169225-0ae8-53bd-947f-07cb81846cb5&t=6b0ace90-28b2-51d9-2951-b1fd35e67dec&l=53), it seems like bazelisk is used and the correct bazel version is pulled when building:

```
2022/04/28 19:31:04 Downloading https://github.com/bazelbuild/bazel/releases/download/6.0.0-pre.20220414.2/bazel-6.0.0-pre.20220414.2-linux-x86_64...
```

Reply:
I think in general the toolchains are pretty stable, so Ideally we won't have to change them much